src/cake.ts

```diff
@@ -20,6 +23,26 @@ export async function runScript(
   }
 }
 
+export async function runProject(
+  csprojPath: string,
+  toolsDir: ToolsDirectory,
+  ...params: CakeParameter[]
+) {
+  const cakeParams = formatParameters(params);
+  const exitCode = await exec(dotnetRun, [
+    '--project', csprojPath,
+    '--no-launch-profile',
+    '--verbosity', 'minimal',
+    '--configuration', 'Release',
+    `--paths_tools="${toolsDir}"`,
+    ...cakeParams
```

Comment:
**question:** `cakeParams` is [going to contain](https://github.com/cake-build/cake-action/blob/5b75644e7b0f8b65d2251f5be6c5d2b9e5e28a66/src/action.ts#L73) the `verbosity` parameter with an argument potentially set by the user. Will its value override the hard-coded `--verbosity minimal` argument?

Reply:
dotnet run and Cake (frosting) arguments need to be separated by `--` 

i.e.
`dotnet run --project build/Build.csproj --no-launch-profile --verbosity minimal --configuration Release -- --paths_tools="tools" --verbosity minimal`

![image](https://github.com/cake-build/cake-action/assets/1647294/d6173c93-e963-48b6-8b52-26dc31a65398)
https://learn.microsoft.com/en-us/dotnet/core/tools/dotnet-run#options

Reply:
Changed the --verbosity flag to use -v so the dotnet run project does not interfere with the cake verbosity setting

Reply:
I ended up removing the flag and the tests pass. There is just a lot of verbosity out of the csproj file. Any suggestions on how to maybe minimize it.

Reply:
@devlead thank you for the suggestion. That separation worked as expected. It allows the verbosity flag used by dotnet run to be separated than the one used by Cake Frosting. 

@ecampidoglio this was the last piece. Is there anything else you have found that needs a second look at,